requirements.in

```
@@ -26,6 +28,7 @@ pre-commit
 polars==1.12.0
 proto-plus>=1.25.0
 psycopg2>=2.9.10
+protobuf>=5.29.2`
```

```suggestion
protobuf>=5.29.2
```

is this a typo, or supposed to have a backtick?